### PR TITLE
from_mot and from_sto methods working fine

### PR DESCRIPTION
**Pull request by romainmartinez**
## Proposed changes

Improved support for opensim file:

- update `from_mot` so that the time vector and acquisition frequency are read directly from the file
- writing `from_sto` with the same characteristics as `from_mot`
- correct a bug in `from_csv` where `time_column = 0` was considered as `time_column = None`

## Types of changes

What types of changes does your code introduce?

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_You can also fill these out after creating the PR. This is simply a reminder of what we are going to look for before merging your code._

- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: pyomeca/analogs.py [hunk 1]
```diff
@@ -73,7 +74,30 @@ def from_2d(m):
 
     @classmethod
     def from_mot(cls, filename):
```
Comment (pariterre) [inline pyomeca/analogs.py]: As a general thought, 
Do you think it would make sense to specify: from_opesim_mot?

File: pyomeca/analogs.py [hunk 2]
```diff
@@ -73,7 +74,30 @@ def from_2d(m):
...
-        return cls.from_csv(filename, header=8, first_column=1, delimiter='\t')
+        mot = cls.from_csv(
+            filename, header=8, first_column=1, time_column=0, delimiter="\t"
+        )
+        mot.get_rate = (1 / (mot.get_time_frames[1] - mot.get_time_frames[0])).round()
+        return mot
+
+    @classmethod
+    def from_sto(cls, filename, endheader_range=20):
```
Comment (pariterre) [inline pyomeca/analogs.py]: Same question!

File: pyomeca/analogs.py (hunk 1 above)
Comment (romainmartinez) [reply pyomeca/analogs.py]: Not really as both formats were created by opensim devs

Comment (pariterre) [reply pyomeca/analogs.py]: My thought was more about: what if another company decides to create files that uses the same names. That may be overthinking though lol

Comment (romainmartinez) [reply pyomeca/analogs.py]: Yeah that __is__ over engineered haha!

**Pull request merged**